lib/teleterm/api/proto/v1/service.proto

```diff
@@ -124,6 +143,91 @@ enum PasswordlessPrompt {
   PASSWORDLESS_PROMPT_CREDENTIAL = 3;
 }
 
+// RequestState represents the state of a request for escalated privilege.
+enum RequestState {
+  // UNSPECIFIED variant exists to allow RequestState to be explicitly omitted
+  // in certain circumstances (e.g. in an AccessRequestFilter).
+  REQUEST_STATE_UNSPECIFIED = 0;
+  // PENDING variant is the default for newly created requests.
+  REQUEST_STATE_PENDING = 1;
+  // APPROVED variant indicates that a request has been accepted by
+  // an administrating party.
+  REQUEST_STATE_APPROVED = 2;
+  // DENIED variant indicates that a request has been rejected by
+  // an administrating party.
+  REQUEST_STATE_DENIED = 3;
+}
+
+message GetAccessRequestsRequest {
+  // cluster_uri is the cluster uri
+  string cluster_uri = 1;
+  // specifcies a specific request id
+  string id = 2;
+  // specifies a username if set
+  string user = 3;
+  // filters for a specific request state
+  RequestState state = 4;
+}
+
+message GetAccessRequestResponse {
+  AccessRequest request = 1;
+}
+
+message GetAccessRequestsResponse {
+  repeated AccessRequest requests = 1;
+}
+
+message DeleteAccessRequestRequest {
+  string cluster_uri = 1;
+  string request_id = 2;
+}
+
+message DeleteAccessRequestResponse {
+  string success = 1;
+}
```

Comment:
Thanks for keeping them organized in the request / response groups, I think that's the way to go.